lib/controller/CountController.js

```diff
@@ -67,18 +67,18 @@ CountController.push({
     const {
       centrelineId,
       centrelineType,
-      dayOfWeek: daysOfWeek,
-      end,
+      daysOfWeek,
```

Comment:
`CountDAO` is a bit odd - `byCentreline` expects to work on a single study type, while `byCentrelineSummary` accepts `studyTypes` as a filter (and looks at all study types if not provided).  in addition, `byCentreline` offers pagination.

in `CollisionDAO`, by contrast, both corresponding methods accept the same set of filters, while `byCentreline` does not offer pagination.

(IMHO this is fine as is, but we should probably document it eventually.)